yarn-project/aztec.js/src/contract/sent_tx.ts

```diff
@@ -74,6 +80,16 @@ export class SentTx {
     return this.pxe.getUnencryptedLogs({ txHash: await this.getTxHash() });
   }
 
+  /**
+   * Gets notes created in this tx.
+   * @remarks This function will wait for the tx to be mined if it hasn't been already.
+   * @returns The requested notes.
+   */
+  public async getNotes(): Promise<ExtendedNote[]> {
+    await this.wait();
+    return this.pxe.getNotes({ txHash: await this.getTxHash() });
+  }
```

Comment:
Agree in that the name is not very good, but I wouldn't dismiss this just on the basis of "teaching devs" since it's easy to miss. At the very least we should clarify it on the jsdoc.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3137